### PR TITLE
Create VPC with PrivateIPs instead of PrivateIpv4Blocks

### PR DESCRIPTION
**Pull request by wenqiq**

### CODE REVIEW

**Review by zhengxiexie — APPROVED** [review]
LGTM

File: pkg/nsx/services/vpc/builder_test.go [hunk 1]
```diff
@@ -80,8 +80,7 @@ func Test_buildNSXLBS(t *testing.T) {
 
 func TestBuildNSXVPC(t *testing.T) {
 	nc := common.VPCNetworkConfigInfo{
-		PrivateIPs: []string{"192.168.1.0/24"},
-		ShortID:    "short1",
+		ShortID: "short1",
```
Comment (dantingl) [inline pkg/nsx/services/vpc/builder_test.go:83]: remove shortID

**Review by dantingl — APPROVED** [review]

**Review by timdengyun — APPROVED** [review]
LGTM

**Comment by timdengyun** [comment]
please squash two commits into one.

**Comment by wenqiq** [comment]
> please squash two commits into one.

We can use the `Squash and merge` button.

**Pull request merged**